Nodejs-Open-Collective.md

```diff
@@ -0,0 +1,35 @@
+# Node.js Open Collective
+
+The project maintains an Open Collective that is used to accept donations.
+
+The funds are to be used for specific efforts as decided by the TSC and
+documented on this page.
+
+## Account details
+
+The account is manged by the TSC. The user id and password
+are shared with TSC members through 1password.
+
+## Disbursements - management
+
+Potential disbursements will be agreed through TSC discussion
+and before approval, documented in an PR in the
+[TSC repository](https://github.com/nodejs/TSC) which adds
```

Comment:
Should it be the admin repo?

Reply:
I don’t know how consistent we are with this, my understanding is that nodejs/TSC would be for stuff internal to the TSC, so folks who are not TSC members can safely unsubscribe from that repo without missing any info relevant to them.